content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -6,47 +6,49 @@ content_template: templates/concept
-Starting in version 1.8, Kubernetes provides a
-[device plugin framework](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/device-plugin.md)
-for vendors to advertise their resources to the kubelet without changing Kubernetes core code.
-Instead of writing custom Kubernetes code, vendors can implement a device plugin that can
-be deployed manually or as a DaemonSet. The targeted devices include GPUs,
-High-performance NICs, FPGAs, InfiniBand, and other similar computing resources
-that may require vendor specific initialization and setup.
+{{< feature-state for_k8s_version="v1.10" state="beta" >}}
+
+Kubernetes provides a [device plugin framework](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/device-plugin.md)
+that you can use to advertise system hardware resources to the
+{{< glossary_tooltip term_id="kubelet" >}}.
+
+Instead of customising the code for Kubernetes itself, vendors can implement a
+device plugin that you deploy either manually or as a {{< glossary_tooltip term_id="daemonset" >}}.
+The targeted devices include GPUs, high-performance NICs, FPGAs, InfiniBand adapters,
+and other similar computing resources that may require vendor specific initialization
+and setup.
+
 {{% /capture %}}
 
 {{% capture body %}}
 
 ## Device plugin registration
 
-The device plugins feature is gated by the `DevicePlugins` feature gate which
-is disabled by default before 1.10.  When the device plugins feature is enabled,
-the kubelet exports a `Registration` gRPC service:
+The kubelet exports a `Registration` gRPC service:
 
 ```gRPC
 service Registration {
 	rpc Register(RegisterRequest) returns (Empty) {}
 }
 ```
+
 A device plugin can register itself with the kubelet through this gRPC service.
 During the registration, the device plugin needs to send:
 
   * The name of its Unix socket.
   * The Device Plugin API version against which it was built.
   * The `ResourceName` it wants to advertise. Here `ResourceName` needs to follow the
     [extended resource naming scheme](/docs/concepts/configuration/manage-compute-resources-container/#extended-resources)
-    as `vendor-domain/resource`.
-    For example, an Nvidia GPU is advertised as `nvidia.com/gpu`.
+    as `vendor-domain/resourcetype`.
+    (For example, an NVIDIA GPU is advertised as `nvidia.com/gpu`.)
 
 Following a successful registration, the device plugin sends the kubelet the
 list of devices it manages, and the kubelet is then in charge of advertising those
 resources to the API server as part of the kubelet node status update.
-For example, after a device plugin registers `vendor-domain/foo` with the kubelet
+For example, after a device plugin registers `hardware-vendor.example/foo` with the kubelet
 and reports two healthy devices on a node, the node status is updated
-to advertise 2 `vendor-domain/foo`.
+to advertise that the node has 2 “Foo” devices installed and available.
```

Comment:
If the devices are shown as typed `hardware-vendor.example/foo`, we should not change it to `Foo` here.

Reply:
I know it's usual to avoid naming vendors. For the sake of this example, perhaps it'd be easier to use the Kubernetes name and English model name for a real device?
It does sound like the “Foo” wording isn't helping so I'll change that, definitely.

(I want the reader to understand what the Kubernetes name maps to in the real world).

Reply:
Revised